pyClient/test_commands/test_erc_token_mixing.py

```diff
@@ -264,36 +188,31 @@ def main() -> None:
         bob_eth_address,
         alice_eth_address,
         charlie_eth_address,
-        mixer_instance.address
+        zeth_client.mixer_instance.address
     )
 
     # Charlie tries to decrypt the notes from Bob's previous transaction.
-    notes_charlie = charlie_wallet.receive_notes(
+    note_descs_charlie = charlie_wallet.receive_notes(
```

Comment:
what does `descs` stand for? description?

Reply:
Yes.  Previously this was just the note data, but now it is a fuller description (including the "address" in the merkle tree) currently required to spend the note.